documentation/documentation_publisher.ts

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable no-console */
```

Comment:
Can use the `print()` utility in this repo that's just a synonym for console.log, that captures the intent of writing to the terminal and avoids needing to deal with the linter.